src/finn/builder/build_dataflow_config.py

```diff
@@ -73,6 +73,7 @@ class DataflowOutputType(str, Enum):
     RTLSIM_PERFORMANCE = "rtlsim_performance"
     BITFILE = "bitfile"
     PYNQ_DRIVER = "pynq_driver"
+    CPP_DRIVER = "cpp_driver"
```

Comment:
Maybe we could consolidate the two output product options into a single "DRIVER" output instead of adding another variation you have to remember whenever switching the options for the driver build step? You already consolidated the build step as "step_make_driver".

Reply:
Ah, I think I understand now, this selects the driver to be actually built? For all the other options, these just decide whether you want to keep the output, which I think is kind of absurd - if I have a step producing some output, I usually want to keep it... But the point is, these options, so far, are not meant to configure what/how to build but only whether to keep the output, hence my confusion. So this is fine...